### PR TITLE
Ignore tests that use ConveRT for now

### PR DESCRIPTION
**Pull request by tabergma**
**Proposed changes**:
As the ConveRT model is not available right now (https://github.com/RasaHQ/rasa/issues/6806), ignore all tests that would download the ConveRT model so that we are still able to merge other PRs.

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: tests/nlu/featurizers/test_convert_featurizer.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 from rasa.nlu.featurizers.dense_featurizer.convert_featurizer import ConveRTFeaturizer
```
Comment (tmbo) [inline tests/nlu/featurizers/test_convert_featurizer.py:12]: can you please add a comment and a reference to the issue why this test is skipped?

File: tests/nlu/test_train.py [hunk 2]
```diff
@@ -101,6 +106,9 @@ def test_all_components_are_in_at_least_one_test_pipeline():
     all_components = [c["name"] for _, p in all_pipelines for c in p]
 
     for cls in registry.component_classes:
+        if "convert" in cls.name.lower():
+            # TODO ignore ConveRT for now as the model cannot be downloaded
```
Comment (tmbo) [inline tests/nlu/test_train.py:110]: please add a link to the github issue here as well

**Review by tmbo — APPROVED** [review]
looks good, thanks a lot for the quick fix. I think it makes sense to mention the issue for the tests that are skipped and the TODO to make sure we remember why we added them

**Comment by howl-anderson** [comment]
Hi @tabergma, this patch also should port back to v1.x, it will help people who still working on v1.x branch.

**Pull request merged**